src/data-access/apiAccess.ts

```diff
@@ -0,0 +1,163 @@
+import { baseUrl, trendUrl, apiKey, genreListUrl, randomMovieURL, personSearchUrl } from './apiPaths';
+import { MovieDTO } from './movieDTO';
+import { GenreDTO } from './genreDTO';
+import { MovieDetailsDTO } from './movieDetailsDTO';
+import { MovieCastDetailsDTO } from './movieCastDetailsDTO';
+import { MovieTrailersDTO } from './movieTrailersDTO';
+import axiosInstance from './axiosInstance';
+import { RandomMovieDTO } from './randomMovieDTO';
+import { PersonDetailsDTO } from './personDetailsDTO';
+import { PersonPopMoviesDTO } from './personPopMoviesDTO';
+import { SearchPersonDTO } from './searchPersonDTO';
+import { SearchMovieDTO } from './searchMovieDTO';
+
+const API_KEY = `${apiKey}`;
+
+export async function getTrendingMovies(): Promise<MovieDTO[]> {
+    try {
+        const response = await axiosInstance.get(trendUrl);
```

Comment:
There is no consistency between the requests that we have here. 

Some of them use urls from apiPaths file, some of them generated here. I'm fine with both approach but I'd like to have consistency. 


The api paths have baseurl but axios already puts it, so it's not necessary to generate complete url and pass it.